Tests/TensorFlowTests/OperatorTests/DatasetTests.swift

```diff
@@ -80,7 +80,7 @@ final class DatasetTests: XCTestCase {
         let scalars = Tensor<Float>(rangeFrom: 0, to: 5, stride: 1)
         let dataset = Dataset(elements: scalars)
         let shuffled = dataset.shuffled(sampleCount: 5, randomSeed: 42)
-        XCTAssertEqual([0, 4, 1, 3, 2], shuffled.map { $0.scalar! })
+        XCTAssertEqual([0, 4, 1, 3, 2], shuffled.map{ $0.scalar! })
```

Comment:
1. The horizontal spacing was already correct. The change made it incorrect. Same for the next few changes.
2. Arguments need to be swapped.

```suggestion
        XCTAssertEqual(shuffled.map { $0.scalar! }, [0, 4, 1, 3, 2])
```